tests/managed_services/mas_debezium/conftest.py

```diff
@@ -0,0 +1,270 @@
+import contextlib
+import logging
+
+import pytest
+import rhoas_kafka_instance_sdk
+from constants import KAFKA_TOPICS_LIST
+from consumer_pod import ConsumerPod
+from ocp_resources.namespace import Namespace
+from ocp_resources.pod import Pod
+from ocp_resources.utils import TimeoutExpiredError, TimeoutSampler
+from ocp_utilities.infra import cluster_resource
+from pytest_testconfig import py_config
+from rhoas_kafka_instance_sdk.api import acls_api, records_api, topics_api
+from rhoas_kafka_instance_sdk.model.acl_binding import AclBinding
+from rhoas_kafka_instance_sdk.model.acl_operation import AclOperation
+from rhoas_kafka_instance_sdk.model.acl_pattern_type import AclPatternType
+from rhoas_kafka_instance_sdk.model.acl_permission_type import AclPermissionType
+from rhoas_kafka_instance_sdk.model.acl_resource_type import AclResourceType
+from rhoas_kafka_instance_sdk.model.config_entry import ConfigEntry
+from rhoas_kafka_instance_sdk.model.new_topic_input import NewTopicInput
+from rhoas_kafka_instance_sdk.model.record import Record
+from rhoas_kafka_instance_sdk.model.topic_settings import TopicSettings
+from rhoas_kafka_mgmt_sdk.exceptions import ApiException
+from rhoas_kafka_mgmt_sdk.model.kafka_request_payload import KafkaRequestPayload
+from rhoas_service_accounts_mgmt_sdk.model.service_account_create_request_data import (
+    ServiceAccountCreateRequestData,
+)
+
+
+LOGGER = logging.getLogger(__name__)
+WAIT_STATUS_TIMEOUT = 120
+
+
+class NoAvailableRegionsError(Exception):
+    pass
+
+
+@pytest.fixture(scope="class")
+def kafka_supported_region(kafka_mgmt_api_instance, rosa_regions):
+    cloud_provider = py_config["cloud_provider"]
+    LOGGER.info(
+        f"Searching for available kafka cloud region under {cloud_provider} cloud provider"
+    )
+    for region_dict in rosa_regions:
+        region_id = region_dict["id"]
+        with contextlib.suppress(ApiException):
+            kafka_mgmt_api_instance.get_instance_types_by_cloud_provider_and_region(
+                cloud_provider=cloud_provider, cloud_region=region_id
+            )
+            return region_id
+
+    raise NoAvailableRegionsError
```

Comment:
```suggestion
    raise NoAvailableRegionsError
```
please add exception message